en/docs/InstallAndSetup/PerfromanceTuningAndTestResults/tuning-performance.md

```diff
@@ -262,13 +236,18 @@ Some general APIM-level recommendations are listed below:
 <tbody>
 <tr class="odd">
 <td><code style="white-space: nowrap;"><br />Synapse global <br />timeout interval</code></td>
-<td><p>Defines the maximum time that a callback waits in the Gateway for a response from the backend. If no response is received within this time, the Gateway drops the message and clears out the callback. This is a global level parameter that affects all the endpoints configured in the Gateway. synapse.global_timeout_interval    </p>
-<p>The global timeout is defined in the <code>                    &lt;API-M_HOME&gt;/repository/conf/synapse.properties                   </code> file. The recommended value is 120000 ms.</p>
-<div class="admonition danger">
-     <p class="admonition-title">Not supported</p>
-     <p>
-     Cannot configure using deployment.toml</p>
-     </div> 
+<td><p>Defines the maximum time that a callback waits in the Gateway for a response from the backend. If no response is received within this time, the Gateway drops the message and clears out the callback. This is a global level parameter that affects all the endpoints configured in the Gateway.</p>
+<p>The global timeout is defined in the <code>                    &lt;API-M_HOME&gt;/repository/conf/deployment.toml                   </code> file. The recommended value is 120000 ms.</p>
+<div class="code panel pdl" style="border-width: 1px;">
+<div class="codeContent panelContent pdl">
+<pre class="java" data-syntaxhighlighter-params="brush: java; gutter: false; theme: Confluence" data-theme="Confluence" style="brush: java; gutter: false; theme: Confluence"><code>
```

Comment:
Are these <div> and <pre> tags valid? The fact that it states Confluence means that this is some formatting that has got copied over from Confluence during the Confluence to Git doc migration process